.devcontainer/devcontainer.json

```diff
@@ -29,5 +31,6 @@
     "remoteUser": "vscode",
     "hostRequirements": {
         "memory": "8gb"
-    }
+    },
+    "postCreateCommand": "npm install -g azure-functions-core-tools@4 --unsafe-perm=true"
```

Comment:
is this not available as a dev container feature? I see one listed here: https://containers.dev/features

Reply:
Good call, I missed that.  Will fix.

Reply:
Fixed.